src/renderer/tvgLoader.cpp

```diff
@@ -441,15 +441,31 @@ LoadModule* LoaderMgr::loader(const char* name, const char* data, uint32_t size,
     //TODO: add check for mimetype ?
     if (auto loader = _findFromCache(name)) return loader;
 
-    if (auto loader = _findByType(mimeType)) {
-        if (loader->open(data, size, "", copy)) {
-            loader->hashpath = strdup(name);
-            loader->pathcache = true;
-            ScopedLock lock(key);
-            _activeLoaders.back(loader);
-            return loader;
-        } else {
-            TVGLOG("LOADER", "The font data \"%s\" could not be loaded.", name);
+    if (!mimeType.empty()) {
```

Comment:
if (mimeType == null) mimeType = "ttf"

Reply:
please check now, I ignored the mimetype and force to use the ttf loader